content/metrics/metrics-11/contents.lr

```diff
@@ -0,0 +1,15 @@
+title: But I noticed that the last data point went up/down a bit since I last
+looked a few hours ago.  Why is that?
+---
+key: 11
+---
+description:
+
+The reason is that we publish user numbers once we're confident enough
+that they won't change significantly anymore.  But it's always possible
+that a directory reports data a few hours after we were confident enough,
+but which then slightly changed the graph.
+---
+seo_slug: metrics
```

Comment:
1. It would be nice to have a short URL, for example, graph-data-point.. So you will need to rename this question directory.
2. Change the seo_slug and _slug.